src/style.css

```diff
@@ -1,3 +1,79 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  background-color: #f5f5f5;
+}
+
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+/* Header Styles */
+.header {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 100%;
+  background-color: white;
+  padding: 0 20px;
+}
+
+.logo img {
+  display: block;
+  height: 40px;
+}
+
+/* Navigation Styles */
+.navigation {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+}
+
+.nav-list li:not(:last-child) {
+  margin-right: 40px; /* Adjust spacing between links */
+}
+
+.nav-link {
+  text-decoration: none;
+  color: black;
+  font-size: 12px; /* Adjust according to design */
+  line-height: 15px; /* Match the header height */
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-align: center;
+  position: relative; /* For the ::after pseudo-element */
+  transition: color 0.3s ease;
+}
+
+.nav-link:hover {
+  color: var(--blue-color);
+}
+
+.nav-link.is-active {
+  color: var(--blue-color);
+}
+
+.nav-link.is-active::after {
+  width: 35.4px;
+  height: 2.4px;
+  border: 1px solid var(--blue-color);
+  background-color: var(--blue-color);
+  top: 30px;
```

Comment:
Consider using `bottom` instead of `top` for better alignment of the `::after` pseudo-element under the active link.